tests/xtimer_msg_receive_timeout/main.c

```diff
@@ -30,27 +26,27 @@
 #include "xtimer.h"
 #include "timex.h"
 
-#define TEST_PERIOD (100000LU)
+#define TEST_PERIOD     (1*US_PER_SEC) /* 1 sec in US */
```

Comment:
mhm, will test - however, the I better make this `1000LU * US_PER_MS` so it can be adapted more easily?